### PR TITLE
ClientProfile API: Add support for kernel and fuse mount options

### PR DESCRIPTION
**Pull request by nb-ohad**
# Describe what this PR does #

Add support for kernel and fuse mount options as optional fields under the `.Spec.CephFs` field

## Is there anything that requires special attention ##

No

Is the change backward compatible?

Yes

### CODE REVIEW

File: api/v1alpha1/clientprofile_types.go [hunk 1]
```diff
@@ -25,6 +25,15 @@ import (
 type CephFsConfigSpec struct {
 	//+kubebuilder:validation:Optional
 	SubVolumeGroup string `json:"subVolumeGroup,omitempty"`
+
+	// Set mount options to use https://docs.ceph.com/en/latest/man/8/mount.ceph/#options
+	// Set to "ms_mode=secure" when connections.encrypted is enabled in Ceph
```
Comment (Madhu-1) [inline api/v1alpha1/clientprofile_types.go:30]: @nb-ohad can you please remove this comment?

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
@nb-ohad please rebase the PR

File: api/v1alpha1/clientprofile_types.go (hunk 1 above)
Comment (nb-ohad) [reply api/v1alpha1/clientprofile_types.go]: done

**Comment by nb-ohad** [comment]
> @nb-ohad please rebase the PR

@Madhu-1 rebased

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**